packages/firebase_auth/firebase_auth/windows/firebase_auth_plugin.cpp

```diff
@@ -0,0 +1,1007 @@
+#include "firebase_auth_plugin.h"
+
+// This must be included before many other Windows headers.
+#include <windows.h>
+
+#include <chrono>
+#include <thread>
+
+#include "firebase/app.h"
+#include "firebase/auth.h"
+#include "firebase/future.h"
+#include "firebase/util.h"
+#include "firebase_core/firebase_core_plugin_c_api.h"
+#include "messages.g.h"
+
+// For getPlatformVersion; remove unless needed for your plugin implementation.
+#include <VersionHelpers.h>
+#include <flutter/event_channel.h>
+#include <flutter/method_channel.h>
+#include <flutter/plugin_registrar_windows.h>
+#include <flutter/standard_method_codec.h>
+
+#include <future>
+#include <iostream>
+#include <memory>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+#include <string_view>
+#include <unordered_map>
+
+using ::firebase::App;
+using ::firebase::auth::Auth;
+
+namespace firebase_auth_windows {
+flutter::BinaryMessenger* FirebaseAuthPlugin::binaryMessenger = nullptr;
+
+// static
+void FirebaseAuthPlugin::RegisterWithRegistrar(
+    flutter::PluginRegistrarWindows* registrar) {
+  auto plugin = std::make_unique<FirebaseAuthPlugin>();
+
+  FirebaseAuthHostApi::SetUp(registrar->messenger(), plugin.get());
+  FirebaseAuthUserHostApi::SetUp(registrar->messenger(), plugin.get());
+
+  registrar->AddPlugin(std::move(plugin));
+
+  binaryMessenger = registrar->messenger();
+}
+
+FirebaseAuthPlugin::FirebaseAuthPlugin() {}
```

Comment:
Can you include "firebase/log.h" and in the constructor call firebase::SetLogLevel(firebase::kLogLevelVerbose); to see if the firebase app is already created?

Reply:
I get the following issue:
```
auth_request.cc Line 50
Expression: !(sdk.empty() || version.empty())
```